schema/layer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "/layer",
+  "type": "object",
+  "properties": {
+    "layer": { "type": "string" },
+    "count": { "type": "number" },
+    "geometry": { "type": "string" },
```

Comment:
Yep! Done in 3c56c5bee64af689432b48959ad3baf32750eaa3 with an enum:

```
"enum": [ "Point", "Line", "Polygon" ]
```

Are those all of the possible types?